### PR TITLE
0.18.0

### PR DESCRIPTION
**Pull request by enricop89**
[Update]: update SDKs iOS to `2.20.0` and Android to `2.20.1` 
[Fix]:  fix crash in OTPublisherLayout when sessionId is null
[Fix]:  Update uuidv4 module
[Fix]:  `OTCustomAudioDrive` added file in `project.pbxproj`

### CODE REVIEW

**Comment by enricop89** [comment]
Holding because `uuid v8` is broken on React Native: https://github.com/uuidjs/uuid/issues/375

File: package.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
     "axios": "^0.21.1",
     "prop-types": "^15.7.2",
     "underscore": "^1.13.0",
-    "uuid": "^8.3.2"
+    "uuid": "^3.4.0"
```
Comment (abdulajet) [inline package.json:31]: was the downgrade intentional?

**Review by abdulajet — APPROVED** [review]
lgtm

**Review by ggoldens — APPROVED** [review]
LGTM

**Pull request merged**